specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json

```diff
@@ -1463,7 +1511,7 @@
           {
             "in": "body",
             "name": "patch",
-            "description": "Model of worker properties to be patched. See also: https://datatracker.ietf.org/doc/html/rfc7386",
+            "description": "Model of worker properties to be created or patched. See also: https://datatracker.ietf.org/doc/html/rfc7386",
```

Comment:
Patch that supports both create and update should define both 200 and 201 response codes.

Reply:
This has been corrected.